docs/DeveloperGuide.md

```diff
@@ -917,82 +984,106 @@ testers are expected to do more *exploratory* testing.
 
-   2. Double-click the jar file Expected: Shows the GUI with a set of sample elderlies. The window size may not be optimum.
+   2. Double-click the jar file <br>
+      Expected: Shows the GUI with a set of sample elderlies. The window size may not be optimum.
 
 2. Saving window preferences
 
    1. Resize the window to an optimum size. Move the window to a different location. Close the window.
 
-   2. Re-launch the app by double-clicking the jar file.<br>
+   2. Re-launch the app by double-clicking the jar file. <br>
        Expected: The most recent window size and location is retained.
 
-### Adding an elderly
+### Adding a elderly
 
 1. Adding an elderly to NurseyBook
 
-    1. Test case: `addElderly en/Khong Guan a/80 g/M r/201 nn/Gong Kuan rs/Brother p/91234567 e/guanbro@gmail.com addr/London Street 11` <br>
-       Expected: New elderly with the parameters is created. His name is Khong Guan, 80 years old, male, in room number 201, with his Nok's (brother) name Gong Kuan, phone number 91234567, email guanbro@gmail.com and address London Street 11.
-    2. Test case: `addElderly en/John a/77 g/M r/420 t/diabetes nn/Timothy rs/Son`<br>
-       Expected: New elderly with the parameters is created. His name is John, 77 years old, male, in room number 420, has a tag 'diabetes', with his Nok's (son) name Timothy.
+   1. Test case: `addElderly en/Khong Guan a/80 g/M r/10` <br>
+      Expected: Display (switches to) the full list of elderly added to NurseyBook. New elderly with parameters is created.
+   
+   2. Test case: `addElderly en/Alice John a/85 r/2 g/F nn/Mary John rs/Sister p/91234567 e/mj@example.com addr/London Street 11 t/diabetes` <br>
+      Expected: Display (switches to) the full list of elderly added to NurseyBook. New elderly with parameters is created.
+   
+   3. Test case: `addElderly en/Sharon Lim a/50 r/20 g/F nn/John Lim rs/Husband`
+      Expected: Display (switches to) the full list of elderly added to NurseyBook. New elderly with parameters is created.
 
-   Invalid commands to try (Error details shown in the status message):
-    1. Invalid index >= size of task list or <= 0: `editTask 5` or `editTask -1`
-    2. Added name is same as another elderly `editElderly 1 en/Alice` (assuming there is another Alice in the elderly database)
+   4. Test case: `addElderly en/khong guan a/55 g/M r/15` <br>
+      Expected: No elderly is added, and the display within NurseyBook stays the same (i.e. If the current display is a list of tasks, it stays on the same list of tasks.). Error details shown in the status message. <br>
+      Note: This test case must be executed only after you have executed test case 1.
+   
+   5. Invalid commands to try (Error details shown in the status message):
+      * Invalid index >= size of task list or <= 0: `editTask 5` or `editTask -1`
+      * Added name is same as another elderly `editElderly 1 en/Alice` (assuming there is another Alice in the elderly database)
```

Comment:
Perhaps it will be better to put these test cases under `editTask` and `editElderly` when they are added?